Misc/NEWS.d/next/Library/2023-01-14-12-58-21.gh-issue-101015.stWFid.rst

```diff
@@ -0,0 +1,2 @@
+Fix :class:`typing.ForwardRef` evaluation of ``'*tuple[...]'`` string. It
```

Comment:
This doesn't mention that the bug was also encountered with `*tuple[...]` even if it wasn't a forward reference; could you add that?